### PR TITLE
Merge main 4162c5

### PR DESCRIPTION
**Pull request by julien-deramond**
- [x] https://github.com/twbs/bootstrap/commit/0444d2cfe53390d6eaf86e8a6f6976309bb0ad25
  - Note: depending on the order of merges https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1817/commits/495de3af4a668ceb7a510ac957b880be032b39e2 might have an impact on  https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1763 @MewenLeHo 
- [x] https://github.com/twbs/bootstrap/commit/93c09e703494575d2025c9a93356e523f0c4df4c
- [x] https://github.com/twbs/bootstrap/commit/10bcccbc31acd1ad73ea6b7fd7cf86d802223450
- [x] https://github.com/twbs/bootstrap/commit/30ed745afbf6ac50056d1dc77206521677172963
- [x] https://github.com/twbs/bootstrap/commit/454720b2d2a7291de8c86ee223685b2d12885b93
- [x] https://github.com/twbs/bootstrap/commit/69aaaf35a236afc7295429e33f1644bfb66546e2
- [x] https://github.com/twbs/bootstrap/commit/7f75fa47de6afb8fdf07ad9082b66a15a7e29e81
- [x] https://github.com/twbs/bootstrap/commit/d9f8be677325d8060253dd25716764f37dcbd343
- ~~https://github.com/twbs/bootstrap/commit/7c7f6ea933d398382555d9d877790a4d85c4546b~~: there's no `<title>` to remove in Boosted
- [x] https://github.com/twbs/bootstrap/commit/d16efb79b69dddaa4320f59f966a176f3fe9e8fb
- [x] https://github.com/twbs/bootstrap/commit/18fec8bf365e34bc8439af99eddc18ec3e14a8a1
- ~~https://github.com/twbs/bootstrap/commit/f6476508c2d63d1f5d970ae941d04d5338c790f9~~: nothing to do here because we already merged the Dependabot PR
- ~~https://github.com/twbs/bootstrap/commit/ab382104e60b9a883c45ae25b3c38b99849ae72b~~: nothing to do here since we don't have the same core team list
- [x] https://github.com/twbs/bootstrap/commit/fd3da4eb3e459bdbc868d1b9e2bd72a6b9d927d1
- [x] https://github.com/twbs/bootstrap/commit/67f456888979cdf4b7a71f3748f53b970d95ccea
- [x] https://github.com/twbs/bootstrap/commit/2f8e8c9b281a07c8eedd7f6d86e2797928c094aa
- [x] https://github.com/twbs/bootstrap/commit/bc8d2e28b83c09fe115bb9cc886efadcaa0dde32
- [x] https://github.com/twbs/bootstrap/commit/48fb1a4b1b0a39c8fe3c08798f8c9331f31db916
- [x] https://github.com/twbs/bootstrap/commit/d77774da4d4d589d635fad3c0f6bb82a57db6b79
- [x] https://github.com/twbs/bootstrap/commit/f2b8be5417daf1bb19ebb69832c770147c944454
- [x] https://github.com/twbs/bootstrap/commit/b1f86f45ad387df3fad3749bd899c2f22e798f4e
- [x] https://github.com/twbs/bootstrap/commit/f7ec6bca7230d811609ecab4c94163cfc7968e4f
- [x] https://github.com/twbs/bootstrap/commit/6117bf6bf3b61567a6ed98c76936e5a4f2202112
- [x] https://github.com/twbs/bootstrap/commit/b69661c1b2ffce61608e6d1bbfbb181af06c8e9c
- [x] https://github.com/twbs/bootstrap/commit/cffe07401ded16bcca83a6c4e3647a57fcc58ac0
- [x] https://github.com/twbs/bootstrap/commit/4f2a0bbbe46fc44cd032a2735a3b07250c1a5a97 (without regenerating the `package-lock.json`)
- [x] https://github.com/twbs/bootstrap/commit/af34b2b56a3628189ff0997593dc07f349b9d54e
- [x] https://github.com/twbs/bootstrap/commit/471edac3d3cae51d415f03d941af1f27e72c9198
  - I haven't modified in this PR `notify.yml`, `pa11y.yml` and `percy.yml` because it would need specific tests for each one in specific PRs. Not sure it's very useful to take time to do it. 
- ~~https://github.com/twbs/bootstrap/commit/f0ae5cceac1302d51cf62ffb180355bfc3300ae2~~: nothing to do here IMO since we don't have anymore the same triggering actions for all the workflows
- ~~https://github.com/twbs/bootstrap/commit/71e2acef441ca099c6e29a762f86c913d3eacc90~~: nothing to do here IMO since we don't have anymore the same triggering actions for all the workflows
- ~~https://github.com/twbs/bootstrap/commit/3e8c8868eb7e920f96cdfffd2452f2e8591f8390~~: nothing to do here since this score cards workflow has been removed by Bootstrap after
- ~~https://github.com/twbs/bootstrap/commit/9d64a983ee05eb84d0e793e359a7335cafaf3eb6~~: nothing to do here since we don't have any Features example in Boosted
- [x] https://github.com/twbs/bootstrap/commit/6807008159012b9cf370db87dcd804c4cae3a1b5
- [x] https://github.com/twbs/bootstrap/commit/e1315d11f2fb9a43b96dd48bf8f9aaf5305372b2
- ~~https://github.com/twbs/bootstrap/commit/33dd82448c82e8abb5bc3dffefd784210d12c1a8~~: nothing to do here since the modification is already there in Boosted
- [x] https://github.com/twbs/bootstrap/commit/ef75dfdd23eecf5f4427c973232eea17d3b5286c
- ~~https://github.com/twbs/bootstrap/commit/51ba329046530254c5069b47d2cf123046de15c6~~: nothing to do since we don't have this score cards badge
- [x] https://github.com/twbs/bootstrap/commit/6a8d6c0dc7983a88c2377e57d2fb3013662fe257
- [x] https://github.com/twbs/bootstrap/commit/622008310c4c6c022079006e04bcbe9bc097f2e8
- [x] https://github.com/twbs/bootstrap/commit/9d87865d3c07937b5dbde831bdca5f473c226fc5
- [x] https://github.com/twbs/bootstrap/commit/4b0e9695c761d308c65384a40cc15298d8563fe9
- [x] https://github.com/twbs/bootstrap/commit/015f06fd522fabb96778f416ec214130f84fa3bd
- [x] https://github.com/twbs/bootstrap/commit/57228a6b65c7451385e0797b409a637bb042bfc1
- [x] https://github.com/twbs/bootstrap/commit/efce82977789898a875cfb2b824bc99ac52b18d0
- [x] https://github.com/twbs/bootstrap/commit/cf3c1535e63affb87cb905c33979e814fa484778
- [x] https://github.com/twbs/bootstrap/commit/f6687e99b5171da9559f3192aa3397b82adf43e1
- [x] https://github.com/twbs/bootstrap/commit/41f62c5a11d98d49646296f996fa844d05f9514d

### CODE REVIEW

File: site/assets/scss/_component-examples.scss [hunk 1]
```diff
@@ -251,7 +251,7 @@
 
 // Boosted mod
 .simple-list-example-scrollspy .active {
-  color: var(--#{$prefix}link-hover-color);
+  color: var(--bs-link-hover-color);
```
Comment (louismaximepiton) [inline site/assets/scss/_component-examples.scss:254]: It seems like there is also in `site\assets\scss\_anchor.scss`, `site\assets\scss\_clipboard-js.scss`, `site\assets\scss\_search.scss` and `site\assets\scss\_tarteaucitron.scss` where we need to change these values

**Review by louismaximepiton — CHANGES_REQUESTED** [review]
Is there a title to remove in `site\layouts\partials\docs-subnav.html`, `site\layouts\partials\icons\download.svg` and `site\layouts\partials\icons\github.svg`? following https://github.com/twbs/bootstrap/commit/7c7f6ea933d398382555d9d877790a4d85c4546b
Following https://github.com/twbs/bootstrap/commit/471edac3d3cae51d415f03d941af1f27e72c9198, maybe we can create an issue to track missing workflows ? **[Bootstrap Side]** why not on calibreapp-image ?

File: site/assets/scss/_component-examples.scss (hunk 1 above)
Comment (julien-deramond) [reply site/assets/scss/_component-examples.scss:254]: I was thinking about doing it in the next commits from Bootstrap, but maybe there won't be any regarding this topic. I don't remember so let's do it now; I'll play with the Boosted mods if needed. Done via https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1817/commits/91ac6a9a007c8c72762231252ba411215ba9e8aa

File: site/content/docs/5.2/components/carousel.md [hunk 2]
```diff
@@ -108,7 +110,7 @@ This `button` must immediately follow your carousel and have the custom `data-bs
 'Play' and 'Pause' texts can be changed by modifying `data-bs-play-text` and `data-bs-pause-text` custom attributes.
 
 {{< example >}}
-<div id="carouselExamplePause" class="carousel slide" data-bs-ride="carousel">
+<div id="carouselExamplePause" class="carousel slide">
```
[the remark this answers is not in the document]
Comment (julien-deramond) [reply site/content/docs/5.2/components/carousel.md]: Yep that's better with it you're right! Done via https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1817/commits/0c83bd65f208769d307a1872420b4b021b6ad1ec

**Comment by julien-deramond** [comment]
> Following https://github.com/twbs/bootstrap/commit/471edac3d3cae51d415f03d941af1f27e72c9198, maybe we can create an issue to track missing workflows ? [Bootstrap Side] why not on calibreapp-image ?

Yep it is mentioned in the description for action. I'll do it after the merge.
Regarding Bootstrap I'm pretty sure that's not been forgotten but IDK the exact reason.

**Comment by julien-deramond** [comment]
> Is there a title to remove in site\layouts\partials\docs-subnav.html, site\layouts\partials\icons\download.svg and site\layouts\partials\icons\github.svg? following https://github.com/twbs/bootstrap/commit/7c7f6ea933d398382555d9d877790a4d85c4546b

In `site\layouts\partials\docs-subnav.html` we can remove it safely, good catch. Done via https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1817/commits/ece5b3325216e80ab667b250b74242f877bbfbfb. For `download.svg` and `github.svg` I'd say we need to keep'em for a11y reasons.

**Review by louismaximepiton — APPROVED** [review]
LGTM :rocket:

**Pull request merged**